src/main/java/org/opensearch/neuralsearch/processor/TextChunkingProcessor.java

```diff
@@ -304,7 +305,12 @@ private void recordChunkingExecutionStats(String algorithmName) {
         EventStatsManager.increment(EventStatName.TEXT_CHUNKING_PROCESSOR_EXECUTIONS);
         switch (algorithmName) {
             case DelimiterChunker.ALGORITHM_NAME -> EventStatsManager.increment(EventStatName.TEXT_CHUNKING_DELIMITER_EXECUTIONS);
-            case FixedTokenLengthChunker.ALGORITHM_NAME -> EventStatsManager.increment(EventStatName.TEXT_CHUNKING_FIXED_LENGTH_EXECUTIONS);
```

Comment:
Hi @q-andy , 
as I added the new chunking algorithm called `fixed_char_length`, `EventStatName.TEXT_CHUNKING_FIXED_LENGTH_EXECUTIONS` will be divided into two enums

Reply:
Sure, since we added the text chunking algorithm stats in 3.1 it's okay to split. In the future, post-3.1 we should avoid changing statnames since technically it's a breaking change for the API response.